src/schema.ts

```diff
@@ -6,62 +6,38 @@ import {summary} from 'datalib/src/stats';
 import {inferAll} from 'datalib/src/import/type';
 import * as dlBin from 'datalib/src/bins/bins';
 
-import {BinQuery, EncodingQuery} from './query/encoding';
+import {BinQuery, EncodingQuery, FieldQuery} from './query/encoding';
```

Comment:
This file has a lot of incorrect conflict resolutions.  

You should undo them.  (Maybe copy a file from master and reapply your changes are faster.)